ideascube/mediacenter/tests/test_views.py

```diff
@@ -104,6 +104,21 @@ def test_staff_can_edit_document(staffapp, video):
     assert Document.objects.get(pk=video.pk).title == title
 
 
+def test_staff_detail_page_has_no_edit_delete_link_package_id(staffapp):
+    document = DocumentFactory(package_id="foo")
+    response = staffapp.get(reverse('mediacenter:document_detail',
+                                    kwargs={'pk': document.pk}), status=200)
+    assert "Cannot edit or delete this document." in response.content.decode()
+    assert '<a href="/en/mediacenter/document/{}/edit/">Edit</a>'.format(document.pk) not in response.content.decode()
```

Comment:
This is a (wrong) developer assumption. I know that both links (de)activated by the same if, so I've tested only one link.

But yes, we should test also the delete link.